src/plugins/intel_gpu/src/kernel_selector/tensor_type.cpp

```diff
@@ -880,8 +880,9 @@ NDims WeightsTensor::GetSimpleDims(const std::vector<size_t>& d, WeightsLayout l
     return ret;
 }
 
-WeightsTensor WeightsTensor::TransformIgnorePadding(WeightsLayout l, WeightsType t, size_t g, bool should_split) const {
-    const uint32_t src_channels = ChannelsCount(layout) - (DoesGroupDimExist(layout) ? 1 : 0);
+WeightsTensor WeightsTensor::TransformIgnorePadding(WeightsLayout l, WeightsType t, size_t g, bool should_split, bool deformable) const {
+    bool is_grouped_1d_conv = !deformable && (g > 1) && (OFM().v == g);
```

Comment:
Basically it looks not good because there are so special handling for only is_grouped_1d_conv.
There are some questions.
1. Is the reason for creating the is_grouped_1d_conv check variable because DoesGroupDimExist cannot handle it?
2. Why add deformable condition in is_grouped_1d_conv? and (g > 1) && (OFM().v == g) is enough for 1d group conv? Don't we need to check x or y length?